providers/provider-protocol-reference.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Provider Protocol Reference
+category: 63d7e8bdbf7b4b0e0745e823
+slug: provider-protocol-reference
+---
+
+# Provider Protocol Reference (v2)
+
+All functions decribed here take msgpack-serialized payloads as arguments and
+return values. The key/values to use for these payloads are exactly the ones you
+can find in `fiberplane::models` crate documentation (NOTE: put the docs.rs link
+once fiberplane is open-sourced and published).
+
+## Imported functions
+
+These functions are usable as long as you import `fiberplane_pdk::prelude::*` in
+your library code. These functions exist to allow you to use OS-level features by
+asking the runtime for information.
+
+For Fiberplane Daemon, these functions are implemented in [Provider Runtime
```

Comment:
I think we should refer to it as _the_ Fiberplane Daemon? What do you think, @keturiosakys ?